docs/requisitos/elicitacao/entrevista.md

```diff
@@ -0,0 +1,125 @@
+- O produtor pode plantar apenas um tipo em um talhão
+
+### 4.9 Precisa anotar a data de plantio, data da aplicação do agrotóxico e a data da colheita?
+
+- O aplicativo deve possuir um bom desempenho
+- O aplicativo deve possuir uma boa usabilidade
+
+### 4.10  Precisa ter algum documento de identificação do produtor?  
+
+- O aplicativo deve guardar a identificação do produtor
+
+### 4.12 Poderia falar um pouco mais como funcionam os talhões?
+
+- A propriedade é dividida em diversos setores chamados de talhões
+- Um talhão pode conter apenas um tipo de cultura plantada
+- Em um talhão pode ser plantado diversas culturas em épocas diferentes
+
+### 4.13 Como funcionam as épocas de plantio?
+
+- Um talhão pode conter em média 3 plantios em um ano
+
+### 4.14 Como o produtor informa onde ele plantou determinada cultura?
+
+- Presencialmente, o técnico conhece o campo e as divisões feitas pelo produtor
+
+### 4.15 Como é identificada uma cultura?
+
+- A cultura é separada em espécies mas tratadas pelo nome vulgar
+
+## 5. Requisitos
+
+|ID|Descrição|Tipo de Requisito
+|--|--|--|
+|E01|O aplicativo deve ser de fácil utilização|Requisito Não Funcional|
+|E02|O aplicativo deve ser intuitivo|Requisito Não Funcional|
+|E03|O aplicativo deve possuir textos simples|Requisito Não Funcional|
+|E04|O aplicativo deve fornecer a Caderneta de Campo para rastreabilidade das colheitas|Requisito Funcional|
+|E05|O aplicativo deve fornecer uma interface simples para o cadastro da aplicação do agrotóxico|Requisito Não Funcional|
+|E06|O produtor deve ser capaz de cadastrar seu plantio|Requisito Funcional|
+|E07|O produtor pode escolher em qual talhão cadastrar o seu plantio|Requisito Funcional|
+|E08|O produtor deve ser capaz de cadastrar a aplicação de agrotóxicos|Requisito Funcional|
+|E09|O produtor pode cadastrar quando colheu o plantio|Requisito Funcional|
+|E10|O produtor deve ser capaz de saber o período de carência|Requisito Funcional|
+|E11|O técnico deve ser capaz de supervisionar propriedades|Requisito Funcional|
+|E12|O técnico deve ser capaz de cadastrar culturas|Requisito Funcional|
+|E13|O técnico deve ser capaz de cadastrar agrotóxicos|Requisito Funcional|
+|E14|O técnico deve ser capaz de cadastrar os tipos dos agrotóxicos|Requisito Funcional|
+|E15|O aplicativo deve garantir a rastreabilidade dos plantios|Requisito Não Funcional|
+|E16|O aplicativo deve fornecer a Caderneta de Campo para rastreabilidade das colheitas|Requisito Funcional|
+|E17|O aplicativo deve permitir a rastreabilidade das aplicações de agrotóxicos|Requisito Funcional|
+|E18|O aplicativo deve permitir o cadastro da aplicação do agrotóxico pela foto|Requisito Não Funcional|
+|E19|O aplicativo deve possuir um bom desempenho|Requisito Não Funcional|
+|E20|O aplicativo deve possuir uma boa usabilidade|Requisito Não Funcional|
+<h6 align = "center">Tabela 1: Tabela contendo os requisitos levantados pela entrevista</h6>
+<h6 align = "center">Fonte: Autor</h6>
+
+## 6. Bibliografia
+
```

Comment:
Para referências você pode usar o símbolo ">" no início de cada linha:

Exemplo
> [1] Wiegers, K.. Beatty J. Software Requirements. Third Edition. 2013